### PR TITLE
Swift: Update README.md

### PR DESCRIPTION
**Pull request by geoffw0**
Turned the warning at the top into a warning box.

Made the section headers consistent.

I was going to add a new section, but I realized it belongs somewhere else.  :)

### CODE REVIEW

File: swift/README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
 # Swift on CodeQL
 
-## Warning
+> [!WARNING]
+> The Swift CodeQL package is an experimental and unsupported work in progress.
```
Comment (AlexDenisov) [inline swift/README.md:4]: I think it's no longer the case, I suggest we pick up wording from [this page](https://codeql.github.com/docs/codeql-language-guides/codeql-for-swift/), and maybe adding the link here?

**Review by AlexDenisov — CHANGES_REQUESTED** [review]

File: swift/README.md (hunk 1 above)
Comment (geoffw0) [reply swift/README.md]: Good point.  Updated.

Comment (geoffw0) [reply swift/README.md]: @AlexDenisov I don't think I can merge this until you mark your change request as done.

**Review by AlexDenisov — APPROVED** [review]

File: swift/README.md (hunk 1 above)
Comment (AlexDenisov) [reply swift/README.md]: @geoffw0, approved, I didn't know it would be _that_ blocking, TIL 😅

Comment (geoffw0) [reply swift/README.md]: Thanks.

**Pull request merged**